project/plugins.sbt

```diff
@@ -5,3 +5,7 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
```

Comment:
Because it is very useful. Just one command shows all dependencies that have changed. I personally prefer to keep it in project as it makes it easier for others to check dependency updates if their changes require them.